inference-mistral.py

```diff
@@ -3,6 +3,7 @@
 
 import torch
 import transformers
+import datasets
```

Comment:
Is this used somehow?

You should install the [pre-commit](https://pypi.org/project/pre-commit/) hooks to run [ruff](https://github.com/astral-sh/ruff) locally before committing.  It would catch these kind of issues and fix them for you when you are about to commit new code.

To install:
```
pip install pre-commit
pre-commit install
```
then you can run `ruff .` to see all issues or `ruff . --fix` to fix them. Or run them via pre-commit without commit by `pre-commit run --all-files`. The configs for pre-commit are in https://github.com/valohai/mistral-example/blob/main/.pre-commit-config.yaml